z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -80,53 +81,123 @@ Depending on the type of image registry backend an additional 100 GB volume.
+
+From a Red Hat machine, make sure that the repository origins are all set:
 
 ```sh
 sudo subscription-manager register # if not done already
+sudo subscription-manager list --available  # search for the pool ID $YOUR_POOLID
 sudo subscription-manager attach --pool=$YOUR_POOLID # if not done already
 sudo subscription-manager repos --disable=* # if not done already
 
 ## TODO replace repos to s390x repos
 sudo subscription-manager repos \
   --enable=rhel-8-for-s390x-baseos-rpms \
-  --enable=openstack-16-tools-for-rhel-8-s390x-rpms \
   --enable=ansible-2.8-for-rhel-8-s390x-rpms \
   --enable=rhel-8-for-s390x-appstream-rpms
 ```
 
-Then install the packages:
+## Install required packages
+
+**Requirements:**
+
+* Python
+* Ansible
+* Python modules required in the playbooks. Namely:
+  * openstackclient
+  * openstacksdk
+  * netaddr
+
+```sh
+sudo yum install ansible python3-netaddr jq  python3-devel gcc gcc-c++
+pip3 install openstackclient  #pip3 install broken on error code 1, but it will not affect the openstackclinet installation
+```
+
+**Verify the installation:**
 ```sh
-sudo yum install python3-openstackclient ansible python3-openstacksdk python3-netaddr jq
+[root@XXX ~]# openstack
+(openstack)
 ```
 
-Make sure that `python` points to Python3:
+**Make sure that `python` points to Python3:**
 ```sh
 sudo alternatives --set python /usr/bin/python3
 ```
 
+## Generate an SSH private key and add it to the CIC control node
+
+You can use this key to SSH into the ICIC nodes and Bastion Node. 
+1. Create the SSH Key
+```sh
+ssh-keygen -f ~/.ssh/id_rsa -t rsa
+```
+2. Add your SSH key to ICIC control nodes and bastion nodes.
+```sh
+ssh-copy-id root@<CIC-control-node-ip>
+ssh-copy-id root@<Bastion-Node>
+```
+
+## Connect the CIC client by using command line
+
+1. Copy the `icicrc` file to your user's home directory.
+```sh
+[root@XXX]# scp -r root@<CIC-control-node-ip>:/opt/ibm/icic/icicrc .
+```
+
+2. Copy the `icic.crt` file to your certs directory
+```sh
+scp -r root@<CIC-control-node-ip>:/etc/pki/tls/certs/icic.crt /etc/pki/tls/certs/
+```
+
+3. Run source `icicrc` to set the environment variables.
+```sh
+[root@XXX]# source /opt/ibm/icic/icicrc
+ Please input the username: admin
+ Please input the password of admin:
+```
+
+4. Verify the Red Hat machine is connected to ICIC successfully.
+```sh
+[root@XXX]# openstack project list
++----------------------------------+-------------+
+| ID                               | Name        |
++----------------------------------+-------------+
+| 9c301e5e7b3a4e48a45e21690357c1be | icicvm      |
+| 9deef79a109440c3b84e522296285904 | service     |
+| bb7cf9dc985046048ccd39059ebfd6bc | ibm-default |
++----------------------------------+-------------+
+```
+
 ## OpenShift Configuration Directory
 
+This repository contains Ansible playbooks to deploy OpenShift on IBM® Cloud Infrastructure Center.
+
```

Comment:
We may replace the step here, 

## Download this playbook:
```
$ git clone https://github.com/IBM/z_ansible_collections_samples.git
$ cp -r z_ansible_collections_samples/z_infra_provisioning/cloud_infra_center/ocp_upi ocp_upi
$ cd ocp-upi
```

Reply:
update it.